src/pages/workspace/accounting/intacct/import/SageIntacctImportPage.tsx

```diff
@@ -0,0 +1,147 @@
+import {Str} from 'expensify-common';
+import React, {useMemo} from 'react';
+import ConnectionLayout from '@components/ConnectionLayout';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {clearSageIntacctErrorField, updateSageIntacctBillable, updateSageIntacctSyncTaxConfiguration} from '@libs/actions/connections/SageIntacct';
+import * as ErrorUtils from '@libs/ErrorUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import withPolicy from '@pages/workspace/withPolicy';
+import type {WithPolicyProps} from '@pages/workspace/withPolicy';
+import ToggleSettingOptionRow from '@pages/workspace/workflows/ToggleSettingsOptionRow';
+import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+import ROUTES from '@src/ROUTES';
+import type {SageIntacctConnectiosConfig, SageIntacctMappingValue} from '@src/types/onyx/Policy';
+
+function getDisplayTypeTranslationKey(displayType?: SageIntacctMappingValue): TranslationPaths | undefined {
+    switch (displayType) {
+        case CONST.SAGE_INTACCT_MAPPING_VALUE.DEFAULT: {
+            return 'workspace.intacct.employeeDefault';
+        }
+        case CONST.SAGE_INTACCT_MAPPING_VALUE.TAG: {
+            return 'workspace.accounting.importTypes.TAG';
+        }
+        case CONST.SAGE_INTACCT_MAPPING_VALUE.REPORT_FIELD: {
+            return 'workspace.accounting.importTypes.REPORT_FIELD';
+        }
+        case CONST.SAGE_INTACCT_MAPPING_VALUE.NONE: {
+            return undefined;
+        }
+        default: {
+            return undefined;
+        }
```

Comment:
You don't need the second to last case here, no? You can just rely on the default

Reply:
Good catch!